src/utilities/requestPermission.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable import/prefer-default-export */
+import { PermissionsAndroid } from 'react-native';
+
+/**
+ * Helper method to request permissions during runtime.
+ *
+ * When trying to use native features for API level > 23, permissions may
+ * need to be granted for permissions a part of the dangerous group,
+ * which can't be granted during installation.
+ * See:
+ * https://developer.android.com/training/permissions/requesting
+ * https://facebook.github.io/react-native/docs/permissionsandroid
+ *
+ *
+ * @property {String} permission The permission request, examples below
+ * @property {String} message    A reason for requesting the permission
+ * @return   {Object} {success, permissionType, result, error}
```

Comment:
Is there a reason for returning `permissionType`? It looks like it isn't touched by the method.

Reply:
Mainly as it's a generic method there might be some use out of it!

Reply:
In this case the parameter is essentially `const` right? As the caller will already have the variable and the method is "pure" with respect to that parameter, is it worth returning it and potentially causing confusion?